### PR TITLE
[shared_preferences_foundation] Run swift-format on Swift files

### PR DESCRIPTION
**Pull request by jmagman**
Run `swift-format` linter/formatter on Swift files.

Part of turning on swift-format CI formatting https://github.com/flutter/packages/pull/5928

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift [hunk 1]
```diff
@@ -5,24 +5,24 @@
 import Foundation
 
 #if os(iOS)
-import Flutter
+  import Flutter
```
Comment (jmagman) [inline packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift:8]: The only change in non-example/test files are the whitespace changes in this file.  Does change actually need a pubspec version bump @stuartmorgan?

Comment (jmagman) [reply packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift:8]: Actually there's also removal of a semicolon, so it's not just whitespace.

**Review by hellohuanlin — APPROVED** [review]

File: packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift (hunk 1 above)
Comment (stuartmorgan) [reply packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift:8]: That's also unquestionably a no-op change though in terms of compile results.

I'm in favor of having no version change or CHANGELOG for this, as something that doesn't affect anything but our own development.

**Review by stuartmorgan — APPROVED** [review]
LGTM

File: packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift (hunk 1 above)
Comment (stuartmorgan) [reply packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift:8]: (But I'm also fine with leaving it as-is)

**Pull request merged**